LjinuxRoot/bin/jpkg.py

```diff
@@ -0,0 +1,184 @@
+opts2 = ljinux.based.fn.xarg()
+errored = False
+
+jpkg_version = 0
+
+if len(opts2["w"]) > 1 and opts2["w"][0] == "install":
```

Comment:
Instead of writing an entire code block in if statement.
Try checking the reverse statement.

```
if len(opts2["w"]) <= 1 and opts2["w"][0] != "install":
    return
.... 
boiler plate code here 
```

Reply:
return doesn't work under exec

Reply:
smh